xlators/features/index/src/index.c

```diff
@@ -242,9 +242,13 @@ index_worker(void *data)
 }
 
 static void
-make_index_dir_path(char *base, const char *subdir, char *index_dir, size_t len)
+make_index_dir_path(char *base, const char *subdir, char *index_dir, size_t len,
```

Comment:
nit: I think if you return the result of snprintf(), which is supposed to be the length of index_dir, you can save some strlen() calls later.

Reply:
done